### ISSUE TITLE
Scalafix for migration is not able to find the rules

### ISSUE THREAD

**Issue by camilo-paidy** [http4s/http4s#3793]
I was trying to execute the steps for migrating from [0.18 to 0.20+](https://http4s.org/v0.21/upgrading/):

`sbt ";scalafixEnable; scalafix github:http4s/http4s/v0_20"`

And

`sbt ";scalafixEnable; scalafix github:http4s/http4s/v0_21"`

But it says that the rule is not found. Maybe because `master` is `main`?

```
[error] (service / Compile / scalafix) scalafix.sbt.InvalidArgument: 404 - not found https://raw.githubusercontent.com/http4s/http4s/master/scalafix/rules/src/main/scala/fix/v0_21.scala
[error] (client / Compile / scalafix) scalafix.sbt.InvalidArgument: 404 - not found https://raw.githubusercontent.com/http4s/http4s/master/scalafix/rules/src/main/scala/fix/v0_21.scala
[error] (integration-tests / Compile / scalafix) scalafix.sbt.InvalidArgument: 404 - not found https://raw.githubusercontent.com/http4s/http4s/master/scalafix/rules/src/main/scala/fix/v0_21.scala
[error] (Compile / scalafix) scalafix.sbt.InvalidArgument: 404 - not found https://raw.githubusercontent.com/http4s/http4s/master/scalafix/rules/src/main/scala/fix/v0_21.scala
[error] Total time: 2 s, completed Oct 30, 2020 4:35:54 PM
```

If I try to run it with the direct URL to the file:
`sbt ";scalafixEnable; scalafix https://raw.githubusercontent.com/http4s/http4s/main/scalafix/rules/src/main/scala/fix/v0_21.scala"`

But it is not able to compile it and apply the rules.

Do you know how to fix it and run it?

Thank you,

**Comment by rossabaker** [comment]
Oh, nuts. Does it work with `github:http4/http4s/v0_20?sha=main`?  If so, we need to update the doc.

I am guessing based on what I see [in scalafix](https://github.com/scalacenter/scalafix/blob/e31e5c946a14bb3839ef5c087b660953a3485db8/scalafix-reflect/src/main/scala/scalafix/internal/reflect/GitHubUrlRule.scala#L13).

**Comment by satorg** [comment]
I guess that doc page for v0.21 was simply inherited from v0.20.
To be honest, I completely missed that page previously.

I believe, since we publish the `http4s-scalafix` module since v0.21.5,
it should be possible to call the migration rules according to this doc: [External Rules](https://scalacenter.github.io/scalafix/docs/rules/external-rules.html)

**Comment by rossabaker** [comment]
That's true.  I think that's probably the better thing to document?  Finer control over which version is being upgraded to?
